static/app/views/projectInstall/createProject.tsx

```diff
@@ -153,24 +157,19 @@ export function CreateProject() {
       project,
       alertRuleConfig,
     }: {project: Project} & Pick<FormData, 'alertRuleConfig'>) => {
-      const ruleIds = [];
+      const ruleIds: Array<string | undefined> = [];
```

Comment:
This function ended up a bit more complex than expected - happy to clean it up in a follow-up